style.css

```diff
@@ -0,0 +1,315 @@
+body{
+    background-image: url('women.jpg');
+    background-repeat: no-repeat; /* Set background image to not repeat */
+    background-size: cover; /* Scale the background image to cover the entire viewport */
+    font-family: Arial, sans-serif;
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+}
+
+
+*{
+    
+    margin: 0px;
+    padding: 0px;
+    box-sizing: border-box;
+    
+}  
+
+
+
+
+

```

Comment:
I suggest formatting the code with the VS Code extension Prettier for clean code since there are many empty lines.